main.cpp

```diff
@@ -52,95 +52,103 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: cricket
+//  action 1: the cricket chirps
+cricket.chirp();
+//  action 2: the cricket hops
+cricket.hop();
+//  action 3: the cricket rubs its wings
+cricket.rubWings();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: dog
+//  action 1: the dog barks
+dog.bark();
+//  action 2: the dog sits
+dog.sit();
+//  action 3: the dog wags its tail
+dog.wagTail();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: 
+//  action 1: the firefly hovers in place
+firefly.hoverInPlace();
+//  action 2: the firefly lights up
+firefly.lightUp();
+//  action 3: the firefly goes dim
+firefly.dim();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: person
+//  action 1: this person can walk forward
+person.walkForward();
+//  action 2: this person can smile
+person.smile();
+//  action 3: this person can jump up and down
+person.jumpUpAndDown();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: clock
+//  action 1: the clock ticks
+clock.tick();
+//  action 2: the clock shows the time
+clock.showTime();
+//  action 3: the clock shows the date
+clock.showDate();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: pendulum
+//  action 1: the pendulum swings left
+pendulum.swingLeft();
+//  action 2: the pendulum swings right
+pendulum.swingRight();
+//  action 3: the pendulum stops swinging    
+pendulum.stopSwinging();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: stoplight
+//  action 1: the stoplight turns red
+stoplight.turnRed();
+//  action 2: the stoplight flashes yellow 
+stoplight.flashYellow();
+//  action 3: the stoplight turns green
+stoplight.turnGreen();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: sine wave
+//  action 1: the sine wave oscillates
+sineWave.oscillate();
+//  action 2: the sine wave sweeps right
+sineWave.sweepRight();
+//  action 3: the sine wave inverts polarity
+sineWave.invertPolarity();
 
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: laptop
+//  action 1: the laptop starts up
+laptop.startUp();
+//  action 2: the laptop goes into sleep mode
+laptop.goToSleep();
+//  action 3: the laptop shuts down
+laptop.shutDown();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.